source/dotnet/Library/AdaptiveCards.Rendering.Html/AdaptiveCardRenderer.cs

```diff
@@ -477,40 +482,74 @@ protected static void AddContainerElements(HtmlTag uiContainer, IList<AdaptiveEl
                             }
                         }
 
+                        if (!cardElement.IsVisible)
+                        {
+                            uiElement.Style("display", "none");
+
+                            if (uiSeparator != null)
+                            {
+                                uiSeparator.Style("display", "none");
+                            }
+                        }
+                        else
+                        {
+                            // if it's visible and it's the first element, hide the separator
+                            if (isFirstElement)
+                            {
+                                if (uiSeparator != null)
```

Comment:
change null checks from "element != null" to "element"

Reply:
Yeah you are right! my bad 

---
In reply to: [280227777](https://github.com/Microsoft/AdaptiveCards/pull/2767#discussion_r280227777) [](ancestors = 280227777)